plugins/catalog/src/components/EntityContextMenu/UnregisterEntity.tsx

```diff
@@ -34,7 +35,10 @@ interface UnregisterEntityProps {
   onClose: () => void;
 }
 
-export function UnregisterEntity(props: UnregisterEntityProps) {
+export const UnregisterEntity = forwardRef<
```

Comment:
I'll note that forwardRef [seems to be on its way out entirely](https://react.dev/reference/react/forwardRef) - if you would be able to address that (possibly in a followup), it'd be appreciated.